cmd/root.go

```diff
@@ -79,47 +79,27 @@ func initConfig() {
 	viper.SetConfigType("yaml")
 
 	viper.AutomaticEnv() // read in environment variables that match
+}
 
+func ValidateConfigFile() {
```

Comment:
Maybe move both ValidateConfigFile() and ValidateConfig() functions in the `config` package if they are not called in cobra initConfig anymore.

Reply:
Probably with some refactoring as `configCmd` is used here and it would result in an import loop.